### PR TITLE
doc: Add build.gradle.kts examples

### LINKED ISSUE #229: Provide build.gradle.kts examples
Examples like the following don't work in kts files which should be preferred now.

```
  id 'io.github.takahirom.roborazzi'
```

### PR DESCRIPTION
**Pull request by sanao1006**
## Overview 
Add build.gradle.kts example to the README to make Roborazzi setup easier.

## Screenshot
image

## Related issue
Closes #229

### CODE REVIEW

**Comment by takahirom** [comment]
It's a bit tricky, but could you add this information to the topic markdown and then generate the README?
https://github.com/takahirom/roborazzi/blob/main/docs/topics/build_setup.md

In README
>

**Comment by sanao1006** [comment]
`. /gradlew generateReadme` to generate the markdown, the same README was generated👍

File: docs/topics/build_setup.md [hunk 1]
```diff
@@ -50,6 +51,54 @@ apply plugin: "io.github.takahirom.roborazzi"
...
+```kotlin
+plugins {
+    ...
+    id("io.github.takahirom.roborazzi")
+}
+```
+
+</td></tr>
+
+</table>
```
Comment (takahirom) [inline docs/topics/build_setup.md:101]: Thank you! This document might serve as the first impression of this library. The .kts format has become more standard now. Therefore, I suggest we prioritize the kts format and use a details tag for the build.gradle version. 
How about this?

----
**build.gradle.kts**
<table>
<tr><td>plugins</td><td>buildscript</td></tr>
<tr><td>

Define plugin in root build.gradle.kts

```kotlin
plugins {
  ...
  id("io.github.takahirom.roborazzi") version "[version]" apply false
}
```

Apply plugin in module build.gradle.kts

```kotlin
plugins {
  ...
  id("io.github.takahirom.roborazzi")
}
```

</td><td>

root build.gradle.kts

```kotlin
buildscript {
  dependencies {
    ...
    classpath("io.github.takahirom.roborazzi:roborazzi-gradle-plugin:[version]")
  }
}
```

module build.gradle.kts

```kotlin
plugins {
    ...
    id("io.github.takahirom.roborazzi")
}
```

</td></tr>

</table>
<details>
<summary>
build.gradle version
</summary>
<table>
<tr><td>plugins</td><td>buildscript</td></tr>
<tr><td>

Define plugin in root build.gradle

```groovy
plugins {
  ...
  id "io.github.takahirom.roborazzi" version "[version]" apply false
}
```

Apply plugin in module build.gradle

```groovy
plugins {
  ...
  id 'io.github.takahirom.roborazzi'
}
```

</td><td>

root build.gradle

```groovy
buildscript {
  dependencies {
    ...
    classpath "io.github.takahirom.roborazzi:roborazzi-gradle-plugin:[version]"
  }
}
```

module build.gradle

```groovy
apply plugin: "io.github.takahirom.roborazzi"
```

</td></tr>

</table>

</details>

**Comment by takahirom** [comment]
Looks great! Thank you for your contribution!

**Pull request merged**